clientd3d/intrface.h

```diff
@@ -40,9 +40,8 @@
 #define GRAPHICS_TOP_HEIGHT 24
 
 /* Inventory area */
-//#define INVENTORY_MIN_WIDTH (170 + LEFT_BORDER)
-#define INVENTORY_MIN_WIDTH (185 + LEFT_BORDER + 67)
-#define INVENTORY_MAX_WIDTH (250 + LEFT_BORDER)
+#define INVENTORY_MIN_WIDTH (215 + LEFT_BORDER + 67)
+#define INVENTORY_MAX_WIDTH 9000
```

Comment:
Is this effectively infinite?  If so, maybe just remove whatever code is checking it?

Reply:
I've removed as much of this now as I can. The minimap uses this max value to define a bitmap for the maximum size at the start. I've kept a max minimap value to support this - please let me know if there is a better alternative for that last constant